test/mongo_storage_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule ApplicationRunner.MongoStorageTest do
+  use ApplicationRunner.ConnCase, async: false
+
+  alias ApplicationRunner.MongoStorage
+  alias ApplicationRunner.Errors.BusinessError
+  alias ApplicationRunner.Environment.MongoInstance
+
+  defp setup_mongo(env_id) do
```

Comment:
More tests needs to be done here to make sure that this feature is 100% working as it can cause a lot of problems in the application data.

Reply:
Of course, I had some problems doing the tests so I left them on the side for now, I'll do them as soon as possible!